dartagnan/src/main/java/com/dat3m/dartagnan/utils/options/BaseOptions.java

```diff
@@ -80,11 +80,8 @@ public void parse(String[] args) throws ParseException, RuntimeException {
         programFilePath = cmd.getOptionValue("input");
         targetModelFilePath = cmd.getOptionValue("cat");
 
-        if(cmd.hasOption("target")) {
-            target = Arch.get(cmd.getOptionValue("target"));
-        }
-
-        method = MethodTypes.fromString(cmd.getOptionValue(METHOD_OPTION, INCREMENTAL.toString()));
+        target = Arch.get(cmd.getOptionValue("target", Arch.NONE.toShortString()));
```

Comment:
Can you double check that this is correct? I remember that I looked through the code before and didn't change this cause there was a slight difference between ```target == NONE``` and ```target == null```.

Reply:
None of the way we have to call the verification was using a `null` target
- `Dartagnan` had this code
```
        if(target == null) {
            System.out.println("Compilation target cannot be inferred");
            System.exit(0);
            return;
        }
```
- In the UI there was always a target as default
- For the UnitTest we were always setting this arch.

We now make it explicit that the default `Arch` is `NONE`

Reply:
Ok, great